### DISCUSSION TITLE
Linked Recipes

### DISCUSSION THREAD

**Discussion by Tommatheussen** [mealie-recipes/mealie#1473] [Feature Request]
For a lot of recipes, some steps are duplicated, mainly for example sauces, or specific methods to prepare potatoes etc.
Something I'd like to see is some sort of 'linked recipes', those could be listed in the ingredients or as a separate step into the main recipe itself. From there we could navigate to the other recipe and get that one made, after which we can navigate back to the main one. 
This would also reduce some of the recipes we have used here (as there's a lot of duplication that will be removed), as well as make it easier to experiment with small changes to the linked recipe, updating it basically everywhere at once if we like the change.

**Comment by Tommatheussen** [comment]
To comment on my own request, we can current sort of do this:
Add a custom ingredient to the list, add the URL of the recipe to the Notes, or add the URL to the step instructions itself

**Comment by michael-genson** [comment]
You should be able to do this already actually, both ingredients and steps support HTML.
```
Step 1
---
If you don't have cooked chicken, <a href="https://myrecipes.com/recipe/my-cooked-chicken-recipe">check out our cooked chicken recipe here</a>.
```

**Reply by aibaron** [reply]
Oh that's awesome! Thanks for showing this, I was thinking it should be in the WYSIWYG editor, but I'm fine doing the basic html. Good to know, thanks!
